### PR TITLE
Sergior

### PR DESCRIPTION
**Pull request by telesforociriaco**

### CODE REVIEW

File: aug_2/test_morning/tictactoe/tictactoe.js [hunk 1]
```diff
@@ -8,3 +91,4 @@ function ticTacToe(plays) {
 
   return {};
 }
+>>>>>>> master
```
Comment (CarlosPadilla) [inline aug_2/test_morning/tictactoe/tictactoe.js]: fix conflicts

File: jul_22/Sergio-practica-try-catch.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        }
+    } catch (e) {
+        console.log(e);
+    }
+
+}
+
+
+//3
+>>>>>>> 8f6bfd61a5682a7c678d0a2a21e38b3cb2b5f018
```
Comment (CarlosPadilla) [inline jul_22/Sergio-practica-try-catch.js]: fix conflicts in your files

File: jul_31/date_exercises_sergio.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+    
+    //console.log(diferencia);
+    return palabra;
+    
+
+}
+let obj2 = new Date('2019-11-31T12:01:00.000')
+console.log(toLazyHuman(obj,obj2));
+>>>>>>> d9c026f49f7146ba2996c9bbd47a356d1261ac5f
```
Comment (CarlosPadilla) [inline jul_31/date_exercises_sergio.js]: fix conflicts in your files

File: aug_2/test_morning/tictactoe/tictactoe.js (hunk 1 above)
Comment (telesforociriaco) [reply aug_2/test_morning/tictactoe/tictactoe.js]: listo

File: jul_22/Sergio-practica-try-catch.js (hunk 2 above)
Comment (telesforociriaco) [reply jul_22/Sergio-practica-try-catch.js]: listo

File: jul_31/date_exercises_sergio.js (hunk 3 above)
Comment (telesforociriaco) [reply jul_31/date_exercises_sergio.js]: ok listo

**Pull request closed without merging**